README.md

```diff
@@ -54,12 +54,18 @@ In your `config.xml`, you need to add the following preferences:
     <preference name="SplashScreen" value="foo" />
     <preference name="SplashScreenDelay" value="10000" />
     <preference name="SplashMaintainAspectRatio" value="true|false" />
+	<preference name="SplashHideNavigationBar" value="true|false" />
```

Comment:
Do these do the same thing as "Fullscreen" and "ShowTitle" prefs?
If so, can we make the default values for these be the values of "Fullscreen" and "ShowTitle"?